plugins/offboard/offboard_impl.cpp

```diff
@@ -257,6 +284,61 @@ void OffboardImpl::send_velocity_body()
     _parent->send_message(message);
 }
 
+void OffboardImpl::send_position_local_ned()
+{
+    _mutex.lock();
+    const float yaw = 0.0f;
+    const float yaw_rate = 0.0f;
+          float x   = (float) _position_local_ned.x_m;
+          float y   = (float) _position_local_ned.y_m;
+          float z   = (float) _position_local_ned.z_m;
+    const float vx  = 0.0f;
+    const float vy  = 0.0f;
+    const float vz  = 0.0f;
+    const float afx = 0.0f;
+    const float afy = 0.0f;
+    const float afz = 0.0f;
+    _mutex.unlock();
+
+    const        uint16_t IGNORE_X			= std::isnan(x) << 0;
+    const        uint16_t IGNORE_Y			= std::isnan(y) << 1;
+    const        uint16_t IGNORE_Z			= std::isnan(z) << 2;
+    const static uint16_t IGNORE_VX			= 1 << 3;
+    const static uint16_t IGNORE_VY			= 1 << 4;
+    const static uint16_t IGNORE_VZ			= 1 << 5;
+    const static uint16_t IGNORE_AX			= 1 << 6;
+    const static uint16_t IGNORE_AY			= 1 << 7;
+    const static uint16_t IGNORE_AZ			= 1 << 8;
+    const static uint16_t IS_FORCE			= 0 << 9; // Setpoint is NOT a force setpoint
+    const static uint16_t IGNORE_YAW		= 1 << 10;
+    const static uint16_t IGNORE_YAW_RATE	= 1 << 11;
```

Comment:
This is all very clean, thank you! A nitpick is the tabs, I vote to change it to spaces.